examples/OpenGLWindow/SimpleOpenGL3App.h

```diff
@@ -16,7 +17,7 @@ struct SimpleOpenGL3App : public CommonGraphicsApp
 	class GLInstancingRenderer* m_instancingRenderer;
 	virtual void setBackgroundColor(float red, float green, float blue);
 
-	SimpleOpenGL3App(const char* title, int width,int height, bool allowRetina=true);
+	SimpleOpenGL3App(const char* title, int width, int height, bool allowRetina=true, std::string windowType="default", int renderDevice=-1);
```

Comment:
what is the string windowType doing here?

Reply:
This allows runtime selection of X11 vs. EGL, which allows using the ExampleViewer without recompiling.